### PR TITLE
Added network port for Learn Layer5 in docs

### LINKED ISSUE #1543: [Docs] Add Learn-Layer5's port number to Network Ports section of Meshery Architecture page
**Reqeuest:**
List the network port used by the `Learn Layer5` application in the Network Ports section of the Meshery Docs - https://meshery.layer5.io/docs/architecture#network-ports

_Originally posted by @leecalcote in https://github.com/layer5io/meshery-osm/pull/45#issuecomment-687906509_

---
**Contributor Resources**
- [Meshery documentation site](https://meshery.layer5.io/docs/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by ghost**
**Description**

This PR fixes #1543

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by KarenEfereyan** [comment]
@leecalcote and @kumarabd I am done with this issue. Referencing [this](https://github.com/layer5io/meshery/issues/1543)

File: docs/pages/architecture.md [hunk 1]
```diff
@@ -9,18 +9,23 @@ permalink: architecture
...
 Meshery uses the following list of network ports to interface with its various components:
 
-| Adapter       | Port          |
-| :------------ | :------------ |
-| Meshery REST API | 9081/tcp |
-{% assign adaptersSortedByPort = site.adapters | sort: 'port' -%}
+| Adapter                                        | Port             |
+| :--------------------------------------------- | :--------------- |
+| Meshery REST API                               | 9081/tcp      
+| Learn Layer5 Application                       | 10011 |
```
Comment (leecalcote) [inline docs/pages/architecture.md:26]: @KarenEfereyan it would be good to append `/tcp` here.

Comment (KarenEfereyan) [reply docs/pages/architecture.md:26]: Will do. I wasnt sure if i was to add it.

**Review by leecalcote — APPROVED** [review]
Very good! Thank you, @KarenEfereyan

**Pull request merged**